sdk/java/pom.xml

```diff
@@ -9,14 +9,15 @@
 	<packaging>jar</packaging>
 	<properties>
 		<hadoop.version>3.0.0</hadoop.version>
+		<argLine>-Djdk.net.URLClassPath.disableClassPathURLCheck=true -Djava.library.path=${project.basedir}/../mount/libjfs:${java.library.path}</argLine>
 	</properties>
 	<build>
 		<plugins>
 			<plugin>
 				<artifactId>maven-surefire-plugin</artifactId>
 				<version>2.19.1</version>
 				<configuration>
-					<argLine>-Djdk.net.URLClassPath.disableClassPathURLCheck=true -Djava.library.path=${project.basedir}/../mount/libjfs:${java.library.path}</argLine>
+					<argLine>@{argLine}</argLine>
```

Comment:
?

Reply:
If surefire and jacoco are used in same pom file , argline paramters of jacoco will be overwritted .More details may refer to https://stackoverflow.com/questions/18107375/getting-skipping-jacoco-execution-due-to-missing-execution-data-file-upon-exec.

Reply:
I see, it's better to leave a comment here.

Reply:
ok